### PR TITLE
Added some documentation

### PR DESCRIPTION
**Pull request by nelsonkopliku**
Just some doc.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# How to contribute
+
+I'm really glad you're reading this, because we need volunteer developers to help this project come to fruition.
```
Comment (fabriziosestito) [inline CONTRIBUTING.md:3]: not sure about the `I'm` here

**Review by fabriziosestito — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (nelsonkopliku) [reply CONTRIBUTING.md:3]: LOL, Copy-pasted :sweat_smile:

Comment (nelsonkopliku) [reply CONTRIBUTING.md]: Changed.

**Review by dottorblaster — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# How to contribute
+
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition.
+
+## Submitting changes
+
+Please send a [GitHub Pull Request to Trento](https://github.com/trento-project/web/pull/new/main) with a clear list of what you've done (read more about [pull requests](http://help.github.com/pull-requests/)). When you send a pull request, we will love you forever if you include updates and extensions to the Golang Tests. We can always use more test coverage. Please follow our coding conventions (below) and make sure all of your commits are atomic (one feature per commit).
```
Comment (rtorrero) [inline CONTRIBUTING.md:7]: I think this reference to the Golang Tests are from the old frontend right?

**Review by rtorrero — APPROVED** [review]
Awesome! Thanks for this, it's really appreciated. Just that minor thingy I commented :-)

File: CONTRIBUTING.md (hunk 2 above)
Comment (nelsonkopliku) [reply CONTRIBUTING.md]: reference removed.

**Pull request merged**